front-end/src/models/index.ts

```diff
@@ -38,8 +39,18 @@ export interface YelpBusinessSearchResponse {
 
 export interface Lobby {
   id: string;
-  participants: (User | null)[];
+  participants: User[];
   numberRestaurants: number;
+  restaurants: Restaurant[];
+  votes: Vote[];
+}
+
+export type VoteResult = "yes" | "no";
```

Comment:
Could we just make this a boolean?

Reply:
I wanted it to be like an enum because we might add blacklisting or veto power or stuff in the future.